.rubocop.yml

```diff
@@ -21,6 +24,7 @@ AllCops:
     - script/**/*
     - vendor/**/*
     - tmp/**/*
+  NewCops: enable
```

Comment:
I'm not so sure we should enable new cops without reviewing their effects first..
CI could fail with an offense unrelated to a commit or PR.

Reply:
It was kind of the point fix style issues as early as possible, a PR should embrace our styling guide. No strong opinion on this. It was just easier than to add all Minitest default cops in the Rubocop File, I was hoping it would lead to less maintenance work.

Reply:
> a PR should embrace our styling guide.

I agree. But I'm talking about offenses unrelated to changes in the PR..

Say a contributor has RuboCop 1.8 installed locally and has prepared their pull request after ensuring `script/fmt` reports no offense. However, by the time they *actually opens the pull request* on GitHub, RuboCop has released a v1.9.0 with many new cops that get enabled automatically and report offenses on code not related to the pull request but fails the CI build nevertheless.